product_docs/docs/migration_toolkit/55/07_invoking_mtk/mtk_command_options_in_file/creating_txt_file.mdx

```diff
@@ -45,64 +45,64 @@ And you can combine both approaches, because MTK will combine all lines that hav
 |                          | tables TAB1 <br/> tables TAB2,TAB3              |
 
 !!!note
-   The duplication of options is only supported for the options file. <br/>
-   When using `-tables` in the command line more than one time, only the last parameter is executed. For example, `./runMTK.sh -tables TAB2 -tables TAB3` only includes the _TAB3_ table in the migration.
+   Duplicating options is supported only for the options file. <br/>
+   When using `-tables` at the command line more than one time, only the last parameter is executed. For example, `./runMTK.sh -tables TAB2 -tables TAB3` includes only the `TAB3` table in the migration.
 !!!
 
 ### Use a space or equal sign to provide values
 
-For options that require values (key-value pairs), separate the option from the value with a space or an equal sign (=). Both options are supported.
+For options that require values (key-value pairs), separate the option from the value with a space or an equal sign (=). Both syntaxes are supported.
 
 | Option in CLI    | Supported syntax for the option in file |
 |------------------|-----------------------------------------|
 | `-views all_emp` | views=all_emp                           |
 |                  | views all_emp                           |
 
-### Don't specify an option in the file and command line
+### Don't specify an option in the file and at the command line
 
-Don't include an option in the options file if you're specifying it as a flag in the command line.
+Don't include an option in the options file if you're specifying it as a flag at the command line.
 
-Specifying an option in both the command line and in the text file will cause the migration to fail.
+Specifying an option both at the command line and in the text file causes the migration to fail.
 
 ## Order of processing
 
 Migration Toolkit reads command line options and option files in the order you provide them when running the command.
 
-For example, if you run the following command, MTK first recognizes the `-sourcedbtype oracle` option, then reads the contents of `example.options` in order from top to bottom. Last parameter is always `schema_name`.
+For example, if you run the following command, Migration Toolkit first recognizes the `-sourcedbtype oracle` option, and then reads the contents of `example.options` in order from top to bottom. The last parameter is always `schema_name`.
 
 ```shell
 runMTK.sh -sourcedbtype oracle -optionsFile example.options schema_name
 ```
 
-If you want an option to be executed last, you can either put it at the end of the `example.options` file and add no further options after `-optionsFile` in the command line.
+If you want an option to be executed last, you can either put it at the end of `<options_textfile>` or add no options after `-optionsFile` at the command line.
```

Comment:
This changes the meaning slightly. If you put an option at the end of the `options_textfile` and you want that to run last, then you should not put other options after that at the command line. 

The other option is to not add the option in the `options_textfile`, but add it as a last reference at the command line.

Let me reorganize this part with some indentation to make the two options clearer.